sabnzbd/downloader.py

```diff
@@ -529,24 +576,6 @@ def run(self):
 
             if readkeys or writekeys:
                 read, write, error = select.select(readkeys, writekeys, (), 1.0)
-
-                # Why check so often when so few things happened?
-                if self.can_be_slowed and len(readkeys) >= 8 and len(read) <= 2:
-                    time.sleep(0.01)
-
-                # Need to initialize the check during first 20 seconds
-                if self.can_be_slowed is None or self.can_be_slowed_timer:
-                    # Wait for stable speed to start testing
-                    if not self.can_be_slowed_timer and BPSMeter.do.get_stable_speed(timespan=10):
-                        self.can_be_slowed_timer = time.time()
-
-                    # Check 10 seconds after enabling slowdown
-                    if self.can_be_slowed_timer and time.time() > self.can_be_slowed_timer + 10:
-                        # Now let's check if it was stable in the last 10 seconds
-                        self.can_be_slowed = BPSMeter.do.get_stable_speed(timespan=10)
-                        self.can_be_slowed_timer = 0
-                        logging.debug('Downloader-slowdown: %r', self.can_be_slowed)
-
```

Comment:
Removed because the sleep on idle_count should be enough. From what I understand this part is not necessary for the download throttling to work.